atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/transaction/impl/SnapshotTransaction.java

```diff
@@ -1569,12 +1581,14 @@ protected boolean shouldDeleteAndRollback() {
     @Override
     public final void delete(TableReference tableRef, Set<Cell> cells) {
         putInternal(tableRef, Cells.constantValueMap(cells, PtBytes.EMPTY_BYTE_ARRAY));
+        cells.forEach(cell -> cache.write(tableRef, cell, CacheValue.empty()));
```

Comment:
I can add a method for `cache.delete`, would make this nicer.

Reply:
Yup, that would make sense 👍